### ISSUE TITLE
Curriculum Training  and Dataset

### ISSUE THREAD

**Issue by xingjici** [Canjie-Luo/MORAN_v2#47]
Hi.
    I am reading your original paper but confused with the data sequences .''
We first optimize the ASRN by using regular training samples. The dataset released by Gupta et al. [15]''......''[20] and Gupta et al. [15] offer abundant irregular training samples.''
    I thought that one dataset may offer regular data for ASRN another irregular for MORN. am I right? But I found that both of 2 above are irregular which will feed into the network.
    In the paper, curriculum training include 3 steps:600,000 for MORN;20,000 forASRN;300,000 for e2e.But I fount the code will directly train in a e2e fashion.am I right? when I insert my dataset ,shell I modify the code to have 3 step illustrated in the paper? or directly feed my data into the network ?
Thank you for your time and consideration!

**Comment by Canjie-Luo** [comment]
Thanks for your attention of our paper. Actually the review of the paper took a long time, and we improved our model and released the MORAN v2. That means that our paper -> v1, our released code -> v2. Please use the released code, which is end-to-end trainable.

**Comment by JingLiJJ** [comment]
> Thanks for your attention of our paper. Actually the review of the paper took a long time, and we improved our model and released the MORAN v2. That means that our paper -> v1, our released code -> v2. Please use the released code, which is end-to-end trainable.

Hi. Have you changed the learning rate during end-to-end training, or lr is always 1?

**Comment by Canjie-Luo** [comment]
We decreased the lr to 0.1 after 3 epoches. You can change the strategy according to your task.
